controllers/managedocs_controller_test.go

```diff
@@ -805,6 +811,38 @@ var _ = Describe("ManagedOCS controller", func() {
 				}, timeout, interval).Should(BeTrue())
 			})
 		})
+		When("the ocsInitialization resource is created", func() {
+			It("should patch the ocsInitialization to enable ceph toolbox", func() {
+				ocsInit := ocsInitializationTemplate.DeepCopy()
+				Expect(k8sClient.Create(ctx, ocsInit)).Should(Succeed())
+
+				ocsInitKey := utils.GetResourceKey(ocsInit)
+
+				// Wait for the spec changes to be reverted
+				Eventually(func() bool {
+					Expect(k8sClient.Get(ctx, ocsInitKey, ocsInit)).Should(Succeed())
+					return ocsInit.Spec.EnableCephTools
+				}, timeout, interval).Should(Equal(true))
+			})
+		})
+		When("the ocsInitialization resource is modified", func() {
+			It("should revert the changes in ocsInitialization to enable ceph toolbox", func() {
+				ocsInit := ocsInitializationTemplate.DeepCopy()
+				ocsInitKey := utils.GetResourceKey(ocsInit)
+				Consistently(func() error {
+					return k8sClient.Get(ctx, ocsInitKey, ocsInit)
+				}, timeout, interval).Should(Succeed())
```

Comment:
Why Consistently? did you mean Eventually? if you just need Eventually on a resource you can use `utils.WaitForResource`

Reply:
My bad, simple `Expect` satisfied the need.